spec/controllers/spina/admin/jobs/job_applications_controller_spec.rb

```diff
@@ -2,29 +2,29 @@
 
 module Spina::Admin
   RSpec.describe Jobs::JobApplicationsController, type: :controller do
-    let(:job_application) { FactoryGirl.create :spina_jobs_job_application }
-    let(:job_applications) { FactoryGirl.create_list :spina_jobs_job_application, 3 }
+    let(:job_application) { FactoryBot.create :spina_jobs_job_application }
```

Comment:
Given we are now adding the `FactoryBot::Syntax::Methods` helper, we can call `create` and `build` directly without explicitly calling `FactoryBot.create`